modules/nf-core/miranda/main.nf

```diff
@@ -0,0 +1,50 @@
+process MIRANDA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::miranda=3.3a" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/miranda:3.3a--h779adbc_3':
+        'quay.io/biocontainers/miranda:3.3a--h779adbc_3' }"
+
+    input:
+    tuple val(meta), path(fasta)
```

Comment:
I'm not very familiar with these applications, but is there any other name you could give this input? 
`fasta` is generally used for reference genome fasta files and could cause some confusion?